Esterni/PianoDiQualifica/res/sections/qualita_processo/qualita_processo.tex

```diff
@@ -0,0 +1,219 @@
+    \item \textbf{Standard:} definire lo standard di processo ogni qualvolta sia possibile al fine di perseguire un approccio incrementale;
+    \item \textbf{Formazione:} ciascun componente del gruppo deve possedere un idoneo  livello di preparazione iniziale al fine di minimizzare i ritardi nella produzione;
+    \item \textbf{Budget:} è necessario contenere eventuali eccedenze rispetto al costo preventivato;
+    \item \textbf{Calenderizzazione:} assicurare una pianificazione adatta ai compiti da svolgere sulla base delle risorse disponibili al fine di massimizzare l'efficienza della produzione.
+\end{itemize}
+
+\subsubsection{Metriche}
+
+\begin{itemize}
+    \item \glock{Cost Variance};
+    \item \glock{Schedule Variance}.
+\end{itemize}
+
+\subsubsection{Obiettivi}
+\begin{itemize}
+    \item \textbf{Rispetto della pianificazione:} ci si pone l'obiettivo di rispettare quanto pianificato all'interno del \textit{Piano di Progetto v.1.0.0} 
+    ove sono presenti le date di scadenza delle varie attività;
+    \item \textbf{Rispetto del budget:} le variazioni delle riorse a disposizione dovranno essere minimizzate cercando di evitare discostamenti da quanto preventivato;
+    \item \textbf{Rispetto del ciclo di vita:} ogni processo deve seguire le fasi del \textit{Ciclo di Deming};
+    \item \textbf{Versionamento:} i prodotti devono essere tracciati con un numero di versione al fine di indivudare le cause di malfunzionamenti.
+\end{itemize}
+
+
+\subsection{Processo di analisi}
+Questo processo tiene conto di tutte le attività di analisi intraprese nel corso dello sviluppo del prodotto ovvero in primis il documento di \textit{Analisi dei Requisiti}
+ma anche le altre attività di identificazione ed analisi:
+
+\begin{itemize}
+    \item  \textbf{Individuazione dei requisiti:} sulla base di quanto descritto dal proponente nel capitolato d'appalto oltre che nei successivi colloqui;
+    \item \textbf{Preventivo:} sulla base dei requisiti individuati e delle risorse a disposizione è necessario produrre un preventivo del costo finale del progetto;
+    \item \textbf{Analisi ed identificazione dei rischi:} è fondamentale identificare ed analizzare i potenziali rischi e prevedere azioni di mitigazione degli stessi.
+\end{itemize}
+
+\subsubsection{Metriche}
+
+Si identificano le seguenti metriche per la verifica dei processi di analisi, i risultati potrebbero non essere disponibili al presente stato di avanzamento del progetto
+in quanto non valorizzabili o scarsamente quantificabili.
+
+
+
+\subsubsection{MPR-DOC4: Cost Variance (CV)} \label{_MPR-DOC4}
+Indica se il valore realmente maturato è maggiore, uguale o minore rispetto al costo effettivo.
+\begin{itemize}
+    \item \textbf{Formula}: $CV = BCWP - ACWP$;
+    \item \textbf{Risultato}: se $CV > 0$ significa che il progetto produce con maggior efficienza (minor costo) rispetto a quanto pianificato, viceversa se negativo.
```

Comment:
Le metriche sono da descrivere qui o nelle norme? In ogni caso mancano le descrizioni delle sigle nelle formule

Reply:
tutto nelle norme, rimuovo dal PdQ tutte le descrizioni delle metriche